Makefile

```diff
@@ -185,6 +185,8 @@ dupl:
 
 .PHONY: check
 check:
+	@echo "checking for missing license headers"
+	@! git grep -LE '^// (Copyright|Code generated by)' '*.go'
```

Comment:
why deviate from the convention of separating arguments from file names with `--`?